liveblog-embed/resources.service.js

```diff
@@ -91,7 +91,14 @@
     Posts.$inject = ['$resource', 'config', 'users', 'srcSet', 'fixProtocol'];
     function Posts($resource, config, users, srcSet, fixProtocol) {
         function _completeUser(obj) {
-            if (obj.commenter) {
+            if (obj.syndication_in && obj.mainItem) {
```

Comment:
I think that this logic may be a little wrong, as if the post has been syndicated and edited it will evaluate to true, but the displayed user in this case should actually be the last user who edited it, not the producer_blog_title